ml_enabler/models/ml_model.py

```diff
@@ -46,17 +47,24 @@ class MLModel(db.Model):
 
     id = db.Column(db.Integer, primary_key=True)
     created = db.Column(db.DateTime, default=timestamp, nullable=False)
-    name = db.Column(db.String)
+    name = db.Column(db.String, unique=True)
     source = db.Column(db.String)
     dockerhub_url = db.Column(db.String)
     dockerhub_hash = db.Column(db.String)
     predictions = db.relationship(Prediction, backref='ml_models',
                                   cascade='all, delete-orphan', lazy='dynamic')
 
-    def create(self):
+    def create(self, ml_model_dto: MLModelDTO):
```

Comment:
Is this supposed to have a colon? Is requiring an explicit type as some sort of DB property? I actually don't know, haven't seen this in python.

 Can you add a few works about the 2nd param

Reply:
This a type hinting convention in Python https://docs.python.org/3/library/typing.html